test/binCases/watch/info-verbosity-verbose/info-verbosity-verbose.test.js

```diff
@@ -22,12 +48,62 @@ test("info-verbosity-verbose", async(done) => {
 		"--info-verbosity",
 		"verbose"
 	]);
-	const { stdout, stderr } = result;
-	expect(stdout).toEqual(expect.anything());
-	expect(stdout).toContain("Compilation");
-	expect(stdout).toContain("webpack is watching the files…");
-
-	expect(stderr).toHaveLength(0);
-	expect(stdout).toMatchSnapshot();
-	done();
+
+	// Watch mode with --info-verbosity verbose does not spit the output in one go.
+	// So we need to keep a track of chunks output order
+	// 1. Compilation starting...
+	// 2. webpack is watching the files...
+	// 3. Compilation  finished
+	// 4. Hash and other info
+	// 5. Compilation starting...
+	// 6. Compilation  finished
+	// 7. Hash and other info
+	var chunkNumber = 0;
+	var hash1, hash2;
+
+	webpackProc.stdout.on("data", data => {
```

Comment:
This is great, abstract this and use this for all watch tests

Reply:
Thank you @evenstensberg !
This pattern is used in all watch tests except `info-verbosity-off`. I will modify that to use this pattern.

Reply:
with switch cases too? I didn't see :(

Reply:
Yes yes 😉

Reply:
okay great, then abstract the common functions and you're good